Add docs for Sanix integration (#30587)
home-assistant/home-assistant.github.io: source/_integrations/sanix.markdown

---
title: Sanix
description: Instructions on how to integrate Sanix devices within Home Assistant.
ha_category:
  - Switch

> ha_release: 2024.1

Reviewer: ```suggestion
ha_release: 2024.5
```

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@tomaszsluszniak'
ha_domain: sanix
ha_platforms:
  - sensor
ha_integration_type: integration
---

The Sanix {% term integration %} allows you to get water/sewage level sensor measurements from the Sanix devices made by [BIT Complex](https://bitcomplex.pl/).

## Pre-requisites

> To set up the {% term integration %} go to the [Sanix dashboard](https://sanix.bitcomplex.pl) and sign in to your account.

Reviewer: ```suggestion
To set up the {% term integration %}, go to the [Sanix dashboard](https://sanix.bitcomplex.pl) and sign in to your account.
```

> Go to the `Help` page, and `System version` - there you will find the API token.

Reviewer: ```suggestion
Not anyone can pull your data from the API. You need to authenticate by using an API token. To retrieve your API token, on the [Sanix dashboard](https://sanix.bitcomplex.pl), go to the **Help** page, and under **System version**, copy your API token. You will need this during setup in Home Assistant.
```
Something like this? Please change if I got this wrong.

Author: Yes, sounds great

---
title: Sanix
description: Instructions on how to integrate Sanix devices within Home Assistant.
ha_category:
  - Binary sensor
  - Switch
ha_release: 2024.1
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@tomaszsluszniak'
ha_domain: sanix
ha_platforms:
  - sensor
  - binary_sensor
ha_integration_type: integration
---

> The **Sanix** {% term integration %} allows you to get water/sewage level sensor measurements from the `Sanix` devices made by [BIT Complex](https://bitcomplex.pl/).

Reviewer 2: ```suggestion
The Sanix {% term integration %} allows you to get water/sewage level sensor measurements from the Sanix devices made by [BIT Complex](https://bitcomplex.pl/).
```

> This is an unofficial integration developed by a **Sanix** user.

Reviewer 2: All integrations are community supported, as already listed on our integration page.

```suggestion
```

> ## Setup

Reviewer 2: ```suggestion
## Pre-requisites
```

To set up the {% term integration %} go to the [Sanix dashboard](https://sanix.bitcomplex.pl) and sign in to your account.

As the `Device number` use your `Sanix dashboard` login.

> To get the `Token`, go to the `Help` page, and `System version`.

Reviewer 2: I'm not sure what is meant here?

Author: Just information on how to get the API token for a config flow. I have changed it a bit

> 

Reviewer 2: I expected some kind of include for the config flow here?

<div class="note warning">
The integration was developed and tested only with <b>Sanix S M25</b> device with <b>3.10</b> firmware version.

> </div>

Reviewer 2: Instead of a warning, we could just add a normal paragraph that lists the known supported devices. Other users might be able to extend that list over time.
